docs/user_manual.md

```diff
@@ -17,6 +17,8 @@
 | 0.9.0   | 2022-02-17 | Remco Vermeulen | Finalize scope deviation records                                                                                        |
 | 0.10.0  | 2022-02-28 | Remco Vermeulen | Describe database correctness in the Hazard and Risk Analysis (HARA).                                                   |
 | 0.11.0  | 2022-02-28 | Remco Vermeulen | Updated version to 1.1.0                                                                                                |
+| 0.12.0  | 2022-11-03 | Remco Vermeulen | Add missing deviation analysis report tables to section 'Producing an analysis report'.                                 |
```

Comment:
Can you please add a copy of your design document to the repo? Perhaps `docs/edd` (for engineering design document)

Reply:
Add the design document.